webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/KnoraGroupService.scala

```diff
@@ -47,6 +48,24 @@ case class KnoraGroupService(
       _ <- knoraGroupRepo.save(group)
     } yield group
 
+  def updateGroup(groupToUpdate: KnoraGroup, request: GroupUpdateRequest): Task[KnoraGroup] =
+    for {
+      _ <- request.name match {
+             case Some(value) => ensureGroupNameIsUnique(value)
+             case None        => ZIO.unit
+           }
```

Comment:
Isn't this a oneliner with ZIO.foreach(request.name)(ensureGroupNameIsUnique)?

Reply:
You still have to add .`unit` at the end or use `ZIO.foreachDiscard` instead. Thanks for the hint.

Reply:
@siers [8dce490](https://github.com/dasch-swiss/dsp-api/pull/3195/commits/8dce490303c7d9bee843a08e1c83117b61c9a961)